src/main/java/controller/LottoController.java

```diff
@@ -1,101 +1,92 @@
 package controller;
 
+import domain.BonusNumber;
+import domain.LottoDispenser;
+import domain.WinningLotto;
+import domain.WinningNumber;
+import dto.DrawResultDto;
 import exception.LottoException;
-import service.LottoService;
 import view.InputView;
 import view.OutputView;
 
 public class LottoController {
 
-    private final LottoService lottoService;
-
-    public LottoController(LottoService lottoService) {
-        this.lottoService = lottoService;
-    }
-
     public void startLotto() {
-        buyLotto();
-        settingWinningNumbers();
-        settingBonusNumbers();
-        printWinningResult();
+        LottoDispenser lottoDispenser = buyLotto();
+        displayBuyLottos(lottoDispenser);
+        WinningNumber winningNumber = settingWinningNumbers();
+        WinningLotto winningLotto = settingWinningLotto(winningNumber);
+        displayDrawResult(lottoDispenser.getDrawResult(winningLotto));
     }
 
-    private void buyLotto() {
-        boolean retry = true;
-        while(retry) {
-            retry = buyLottoInput();
-        }
-        displayBuyLottos();
+    private LottoDispenser buyLotto() {
+        LottoDispenser lottoDispenser;
+        do {
+            lottoDispenser = buyLottoInput();
+            return lottoDispenser;
+        } while (lottoDispenser != null);
     }
 
-    private boolean buyLottoInput() {
+    private LottoDispenser buyLottoInput() {
         try{
             String inputBuyLottoMoney = InputView.inputBuyLottoMoney();
-            inputBuyLottoMoney(inputBuyLottoMoney);
+            return new LottoDispenser(inputBuyLottoMoney);
         }catch (LottoException lottoException){
```

Comment:
```suggestion
        } catch (LottoException lottoException) {
```
코딩 스타일 컨벤션도 맞추어주면 좋겠습니다! 이 부분 외에도 전체적으로 정렬을 맞춰주세요~

Reply:
코딩 스타일 컨벤션을 세심하게 확인하지 못했네요 😂

전체적으로 확인해보고 정렬 맞추었습니다!